learning_resources_search/serializers.py

```diff
@@ -469,6 +469,11 @@ class LearningResourcesSearchRequestSerializer(SearchRequestSerializer):
     )
 
 
+class LearningResourcesVectorSearchRequestSerializer(SearchRequestSerializer):
```

Comment:
This shouldn't inherit from SearchRequestSerializer. http://api.open.odl.local:8063/api/v0/schema/redoc/#tag/learning_resources_vector_search/operation/learning_resources_vector_search_retrieve is still showing a bunch of filters that are not yet implemented and some that will not ever be implemented such as dev_mode